postgresql-migration.cabal

```diff
@@ -30,31 +32,32 @@ common common-options
     base >=4.9 && <5
   default-language:
     Haskell2010
+  ghc-options: -Wall
```

Comment:
Consolidate all common options under one common-options - as all of library, tests and executable were using identical options.